app/qml/MerginProjectPanel.qml

```diff
@@ -601,7 +601,7 @@ Item {
         MainPanelButton {
           id: exploreBtn
           width: toolbar.itemSize
-          text: qsTr("Explore")
+          text: qsTr("Public")
```

Comment:
This will be confusing is we rename as we are using the term `Explore` in mergin web and plugin